utils/ebpf/helper_summaries.h

```diff
@@ -10,11 +10,17 @@
 
 extern "C" {
 
-extern void __VERIFIER_error(void);
-extern void __VERIFIER_assert(bool);
+/*extern void __VERIFIER_error(void);*/
+/*extern void __VERIFIER_assert(bool);*/
+void __VERIFIER_error(void) {}
```

Comment:
not sure why we replicate this here. Also, code in header files is a recipe for problems later on...